addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -1223,58 +1223,58 @@ var VideoWidget = MediaWidget.extend({
         }
         var url = embedMatch ? embedMatch[1] : code;
 
-        var query = this._createVideoNode(url, {
+        this._createVideoNode(url, {
```

Comment:
Maybe you can use `await` to not indent all the code in the `then`

Reply:
Done! used 'await' instead of 'then'